### ISSUE TITLE
Error: No module named 'packaging'

### ISSUE THREAD

**Issue by shpolina** [fivetran/great_expectations#4407]
**Describe the bug**

We are running GE using pure python and started to experience an import error `ModuleNotFoundError: No module named 'packaging'` recently.

```
 from great_expectations.data_context.data_context import DataContext
  File "\lib\site-packages\great_expectations\__init__.py", line 7, in <module>    
    from great_expectations.data_context import DataContext
  File "\lib\site-packages\great_expectations\data_context\__init__.py", line 1, in <module>
    from .data_context import BaseDataContext, DataContext, ExplorerDataContext
  File "\lib\site-packages\great_expectations\data_context\data_context.py", line 23, in <module>
    from great_expectations.core.config_peer import ConfigPeer
  File "\lib\site-packages\great_expectations\core\config_peer.py", line 6, in <module>
    from great_expectations.data_context.types.base import BaseYamlConfig
  File "\lib\site-packages\great_expectations\data_context\types\base.py", line 29, in <module>
    from great_expectations.util import deep_filter_properties_iterable
  File "\lib\site-packages\great_expectations\util.py", line 32, in <module>       
    from packaging import version
ModuleNotFoundError: No module named 'packaging'
```
It looks like `packaging` is not explicitly declared as a dependency in [requirements.txt](https://github.com/great-expectations/great_expectations/blob/86583dc5a073f6e769356e39482c43d1e2976359/requirements.txt) and we were
previously getting it thanks to the explicitly declared `ipywidgets` dependency. But `ipywidgets` started to pick
up a later version which no longer brings `packaging` with it.  

1. in clean environment pip install great-expectations
2. import `from great_expectations.data_context import DataContext`
3. Try to invoke the code - and get the error

**Environment (please complete the following information):**
 - Operating System: Windows
 - Great Expectations Version: 0.14.10

**Comment by austiezr** [comment]
@shpolina Thank you for surfacing; we'll investigate ASAP.

**Comment by austiezr** [comment]
Closed by #4447 ; thank you @shpolina !

**Comment by shpolina** [comment]
Thank you very much for the prompt fix.
